frontend/src/components/UserCard.js

```diff
@@ -30,7 +30,7 @@ export default class UserCard extends Component {
 
     const href = `https://opencollective.com/${user.username}`;
 
-    const roleLabel = formatAnchor(user.tier || user.role);
+    const roleLabel = formatAnchor(user.role || user.tier);
```

Comment:
this is what's causing the bug on opencollective.com/tessel. "MEMBER" doesn't  show up because tier is populate as "core-contributor"